.github/workflows/test_periodic.yml

```diff
@@ -0,0 +1,226 @@
+name: ovn-ci-periodic
```

Comment:
Will this file have to always be separate from test.yml? Always get nervous when there is a lot the same or real similar logic in two places and trying to make sure they stay in synch.

Reply:
same comment as in here https://github.com/ovn-org/ovn-kubernetes/pull/1520#discussion_r461001228 

until it is stable, I expect lot of churn in this file, we can do any changes here without impacting the rest of the jobs